### PR TITLE
teams/migration: a vote for sdojjy as the reviewer

### PR DESCRIPTION
**Pull request by overvenus**
As title.

### CODE REVIEW

File: votes/0615-sdojjy-as-migration-reviewer.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# A Vote for sdojjy as Migration Reviewer
+
+## Proposal
+
+[@sdojjy](https://github.com/sdojjy) has been working on `pingcap/ticdc` for several months, he has made contributions to TiCDC sink components:
+
+* [Authored pull requests](https://github.com/pingcap/ticdc/commits?author=sdojjy).
+
+I (@overvenus) hereby nominate @sdojjy as Migration Committer and call for a vote.
```
Comment (lance6716) [inline votes/0615-sdojjy-as-migration-reviewer.md:9]: the json file shows it's reviewer

Comment (amyangfei) [inline votes/0615-sdojjy-as-migration-reviewer.md:9]:
```suggestion
I (@overvenus) hereby nominate @sdojjy as Migration Reviewer and call for a vote.
```

File: votes/0615-sdojjy-as-migration-reviewer.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# A Vote for sdojjy as Migration Reviewer
+
+## Proposal
+
+[@sdojjy](https://github.com/sdojjy) has been working on `pingcap/ticdc` for several months, he has made contributions to TiCDC sink components:
+
+* [Authored pull requests](https://github.com/pingcap/ticdc/commits?author=sdojjy).
```
Comment (amyangfei) [inline votes/0615-sdojjy-as-migration-reviewer.md:7]:
```suggestion
* [Authored pull requests](https://github.com/pingcap/ticdc/pulls?q=is%3Apr+author%3Asdojjy+is%3Aclosed).
```

File: votes/0615-sdojjy-as-migration-reviewer.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+## Deadline
+
+The vote will be open for at least 3 days unless there is an objection or not enough votes.
+
+## Scope
+
+* Team Migration
+
+## Result
```
Comment (lance6716) [inline votes/0615-sdojjy-as-migration-reviewer.md:19]: this section could left empty or

```
(To be updated) Conclude the voting result, including approvals and vetoes, binding and non-binding.
```

**Review by amyangfei — APPROVED** [review]
approve for the result. Cons.

**Review by lance6716 — APPROVED** [review]

**Comment by sunxiaoguang** [comment]
Thanks for your contribution @sdojjy. Congrats!

**Pull request merged**